src/main/java/com/google/firebase/remoteconfig/Template.java

```diff
@@ -73,6 +87,22 @@ public Template() {
     if (templateResponse.getVersion() != null) {
       this.version = new Version(templateResponse.getVersion());
     }
+    this.etag = templateResponse.getEtag();
+  }
+
+  /**
+   * Creates and returns a new Remote Config template from a JSON string.
+   *
+   * @param json A non-null JSON string to populate a Remote Config template.
+   * @return A new {@link Template} instance.
+   * @throws IOException If the input JSON string is not parsable.
+   */
+  public static Template fromJSON(@NonNull String json) throws IOException {
+    checkArgument(!Strings.isNullOrEmpty(json), "JSON String must not be null or empty.");
+    JsonFactory jsonFactory = Utils.getDefaultJsonFactory();
```

Comment:
We should use the JsonFactory that comes from AppOptions

Reply:
Note: Discussed offline and decided to use the default json factory here for local serializations. Added a comment 
in code to clarify.